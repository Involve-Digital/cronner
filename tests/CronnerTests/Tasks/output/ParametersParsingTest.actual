[
	'cronner-task' => 'CronnerTests\\objects\\TestObject - test01',
	'cronner-period' => '5 minutes',
	'cronner-days' => null,
	'cronner-days-of-month' => null,
	'cronner-time' => null,
]
