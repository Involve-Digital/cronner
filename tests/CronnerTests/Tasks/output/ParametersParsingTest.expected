[
	'cronner-task' => 'E-mail notifications',
	'cronner-period' => '5 minutes',
	'cronner-days' => null,
	'cronner-days-of-month' => null,
	'cronner-time' => null,
]
